generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -131,7 +131,7 @@ class FireSimSmallSystemConfig extends Config(
   new WithoutClockGating ++
   new WithoutTLMonitors ++
   new freechips.rocketchip.subsystem.WithExtMemSize(1 << 28) ++
-  new testchipip.WithTSI ++
+  new testchipip.WithDefaultSerialTL ++
```

Comment:
This small config seems like a good place to put a FireSim that test LBWIF. (reduces number of off-chip IOs of the modeled system by a lot)

Reply:
I'm currently uncertain if FireSim configs without the AXI4 backing memory work correctly, since they use a separate channel to write the program binary into DRAM.

Reply:
They don't work correctly, issue opened here: https://github.com/firesim/firesim/issues/627